core/jazz_ui/src/app/pages/login/oss/login.component.ts

```diff
@@ -256,17 +256,11 @@ export class LoginComponent implements OnInit {
                         this.forgot_password = false;
                         this.onChange(e);
                         this.model.password = this.model.verificationCode = '';
-                        let successMsg = this.toastmessage.customMessage('success', 'reset');
+                        let successMsg = this.toastmessage.customMessage('success', 'resetPwd');
                         this.toast_pop('success', 'Success!', successMsg);
-                        
                     }, err => {
                         let error = JSON.parse(err._body);
-                        let errorMessage=this.toastmessage.errorMessage(err, 'reset');
-                        try{
-                            errorMessage = error.message;
-                        } catch(e) {
-                            console.log(e);
-                        }
+                        let errorMessage=this.toastmessage.errorMessage(err, 'resetPwd');
```

Comment:
Consider a condition where toast message for unknown error is not defined in toast message. In such case use the message from API. (Same in other update password condition)

Reply:
Handled by using try catch.